Exploring_Data_NCAR.ipynb

```diff
@@ -0,0 +1,2073 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    ds = cat['CESM_POP_hires_RCP8_5'].to_dask()
</pre><p>Note that this line indicates that you are loading in the RCP8.5 scenario - which is the climate change run of this NCAR model. The "control" is the model run without climate change forcing.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/oceanhackweek/ohw22-proj-gcmfilters/pull/10/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>